nncf/config/schema.py

```diff
@@ -713,19 +713,36 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False
 }
 
+KNOWLEDGE_DISTILLATION_ALGO_NAME_IN_CONFIG = 'knowledge_distillation'
+KNOWLEDGE_DISTILLATION_SCHEMA = {
+    **BASIC_COMPRESSION_ALGO_SCHEMA,
+    "properties": {
+        "algorithm": {
+            "const": KNOWLEDGE_DISTILLATION_ALGO_NAME_IN_CONFIG
+        },
+        "scale": with_attributes(_NUMBER,
+                                 description="Scale parameter for kdloss"),
```

Comment:
not clear why this scale is needed. please describe it in more details

Reply:
Scale is just a FP variable to multiply KDLoss value to.


`return self.scale * reduce(            lambda kd_loss, loss_tensors: kd_loss + self.kdloss_fn(loss_tensors[0], loss_tensors[1]),            zip(orig_model_loss_outputs, compressed_model_loss_outputs), 0)`


In other words:
KDLoss = scale * KDLoss

Reply:
Actually this option is not necessary or frequently used and could be abandoned.

Reply:
Also meant comment for the user in the code.
But if it's not useful we can remove it

Reply:
I'd leave it but rename to something self-explanatory like `kd_loss_weight_in_total_loss`